code/edge/src/androidTest/java/com/adobe/marketing/mobile/ConsentStatusChangeFunctionalTests.java

```diff
@@ -58,6 +56,9 @@ public class ConsentStatusChangeFunctionalTests {
 
 	@Before
 	public void setup() throws Exception {
+		mockNetworkService.reset();
```

Comment:
For all tests:
- how about moving this in the @After 
- also should you call resetTestExpectations instead?